php/ContentSession.php

```diff
@@ -314,6 +313,7 @@ private function getToken(Google_Client $client) {
   }
 
   protected function cacheToken(Google_Client $client) {
+    throw new Exception('This example currently only supports the service account flow..');
     print (str_repeat('*', 40) . "\n");
```

Comment:
Please remove lines 317-324 so we don't demonstrate an unsupported approach + we avoid unreachable code.

Reply:
Also remove the `getToken` method entirely